exasol_integration_test_docker_environment/lib/docker/images/create/utils/file_directory_list_hasher.py

```diff
@@ -155,23 +210,25 @@ def _reduce_hash(self, hashes):
 class PathHasher:
     def __init__(self, hashfunc: str = 'md5',
                  hash_permissions: bool = False,
-                 use_relative_paths: bool = False, ):
+                 use_relative_paths: bool = False,):
```

Comment:
I think, this option now also doesn't matter anymore. It made sense, when path hasher was more general, but now it is very specific to our problem

Reply:
Actually I was unsure if I could remove. I will remove it then, however this mean I will have to double-check all the tests.

Reply:
Now, all path should be relative to the build context root.